### PR TITLE
[JENKINS-50216] - RemotableGoogleCredentials.expiration fix

### PR DESCRIPTION
**Pull request by jglick**
Supersedes #15 much more simply. See my explanation in [JENKINS-50216](https://issues.jenkins-ci.org/browse/JENKINS-50216). CC @oleg-nenashev

### CODE REVIEW

File: src/main/java/com/google/jenkins/plugins/credentials/oauth/RemotableGoogleCredentials.java [hunk 1]
```diff
@@ -136,7 +137,7 @@ public Credential getGoogleCredential(
   /**
    * The time at which the accessToken will expire.
    */
-  private final DateTime expiration;
+  private final long expiration;
```
Comment (oleg-nenashev) [inline src/main/java/com/google/jenkins/plugins/credentials/oauth/RemotableGoogleCredentials.java]: Maybe you also want to rename the field just to cause there are usages outside the case we know about

**Comment by oleg-nenashev** [comment]
OTOH I would advice to recover the roundtrip and "data migration" tests from #15 . The latter one would need to be adjusted to just ensure that the data loads and that the key is invalidated.

**Comment by jglick** [comment]
AFAICT—based on my interpretation of the bug and the plugin’s expected usage of this class as described in JIRA—there is no reason to rename the field, no reason to have tests covering XStream round-tripping or data migration, and no reason to mark the plugin as incompatible:

* Normally the class is serialized only over Remoting, which would always be using bytecode from the currently installed plugin version (never persisted).
* The XStream occurrence is one-way (serialized for a checksum, never deserialized), and given other bugs in the plugin, what it serializes to is completely irrelevant.

**Comment by ryangood** [comment]
Any update on when this might be released? for some reason im still running into the issue with the latest commit of .6

**Comment by oleg-nenashev** [comment]
@astroilov thanks! Let me know once it is released, I will update the documentation.

P.S: I still think that some roundtrip tests from #15 would not hurt, because currently the plugin has almost no tests with `JenkinsRule`. But I will unlikely have time to recover it. Created https://issues.jenkins-ci.org/browse/JENKINS-51076 as a follow-up

**Comment by oleg-nenashev** [comment]
@astroilov Please let us know if you have any issues with the release. The PR is merged to master, but it is not available to users so far

**Comment by astroilov** [comment]
Sorry, I was out of office the last few days. This is now released.

**Comment by oleg-nenashev** [comment]
Thanks a lot!

**Pull request merged**